TOOLS_OVERVIEW.md

```diff
@@ -0,0 +1,254 @@
+## Overview of CProver Tools
+
+This document provides a brief overview of the various tools in
+the CProver project. The goal is as to provide a lightweight reference
+with links to detailed documentation on each of the tools.
+
+The tools in the CProver project are as follows:
+- [cbmc](#cbmc)
+- [goto-analyzer](#goto-analyzer)
+- [goto-cc (goto-gcc, goto-ld)](#goto-cc)
+- [goto-diff](#goto-diff)
+- [goto-harness](#goto-harness)
+- [goto-instrument](#goto-instrument)
+- [janalyzer](#janalyzer)
+- [jbmc](#jbmc)
+- [jdiff](#jdiff)
+- [memory-analyzer](#memory-analyzer)
+- [smt2_solver](#smt2_solver)
+- [symtab2gb](#symtab2gb)
+- [Developer Utilities](#developer-utilities)
+    - [java-unit](#java-unit)
+    - [unit](#unit)
+    - [others (converter, driver, file_converter, java-converter)](#others)
```

Comment:
This is great, and makes me wonder whether this should be linked (or included in?) the top-level README file so that people accessing the repository see it right away?

Reply:
Added link in README.md